python/pyarrow/parquet.py

```diff
@@ -1471,6 +1471,18 @@ def _make_manifest(path_or_paths, fs, pathsep='/', metadata_nthreads=1,
     return pieces, partitions, common_metadata_path, metadata_path
 
 
+def _is_local_file_system(fs):
+    import fsspec
```

Comment:
The reason for the failing tests is this import, since `fsspec` is not a required depedendency, we can't assume we can import it here as such.